### PR TITLE
Angular: Add back-compat method to find options (styles) in angular.json

### PR DESCRIPTION
**Pull request by ThibaudAV**
TODO

Issue:

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: app/angular/src/server/framework-preset-angular-cli.ts [hunk 1]
```diff
@@ -124,3 +130,65 @@ async function getBuilderOptions(
...
+ * only work for angular.json with [defaultProject].build or "storybook.build" config
+ */
+async function getLegacyDefaultBuildOptions(options: PresetOptions) {
+  if (options.angularBrowserTarget !== undefined) {
+    // Not use legacy way with builder (`angularBrowserTarget` is defined or null with builder and undefined without)
+    return {};
+  }
+
+  logger.info(`=> TODO ~ legacy way to get default options`);
+  logger.info(`=> TODO ~ Add deprecation warning and ex for builder use ? `);
```
Comment (kroeder) [inline app/angular/src/server/framework-preset-angular-cli.ts:146]: lets remove those and merge the PR to hotfix the current issue and improve the situation afterwards
```suggestion
```

**Comment by kroeder** [comment]
I tested it in my project that had this issue and it fixed it. Everything works as expected

File: app/angular/src/server/framework-preset-angular-cli.ts (hunk 1 above)
Comment (Tim-arts) [reply app/angular/src/server/framework-preset-angular-cli.ts:146]: Typo here: "and wild be removed on Storybook 7.0"

**Comment by shilman** [comment]
@ThibaudAV @kroeder I did a little cleanup on the file and will merge/release an a 6.5 alpha. Once we get a few people testing it successfully we can patch the change back to 6.4. Sorry for meddling here but I feel this is time sensitive since people are upgrading to 6.4 now. 🙈

**Comment by ThibaudAV** [comment]
Ok I will add the message of deprecation of this code for v7 and the info in the MIGRATION.md in another PR

**Pull request merged**